custom-fonts.md

```diff
@@ -1,13 +1,19 @@
 ---
 layout: page
-title: Using Custom Fonts
+title: Using Fonts
 
 permalink: /custom-fonts/
 ---
 
-You can use custom fonts in your Flutter application by including them in your
-`pubspec.yaml` file under the `fonts` heading in your `flutter` section:
+## Using fonts
 
+Using fonts in your Flutter application is done in two steps. 
+First include them in your `pubspec.yaml`, 
+second declare the use in `TextStyle` property.
+
+### Example
```

Comment:
Maybe use header names that detail the intention of the step.

For example: "Declaring in the font asset"